modules/nw-ingress-gateway-api-enable.adoc

```diff
@@ -0,0 +1,237 @@
+router-default             2/2     2            2           6h4m
+----
+
+. Create a secret by using the default certificate by running the following command:
++
+[source,terminal]
+----
+$ oc -n openshift-ingress create secret tls gwapi-wildcard --cert=wildcard.crt --key=wildcard.key
+----
+
+. Get the domain of the Ingress Operator by running the following command:
++
+[source,terminal]
+----
+$ DOMAIN=$(oc get ingresses.config/cluster -o jsonpath={.spec.domain})
+----
+
+. Create a `Gateway` object:
+
+.. Create a YAML file, `example-gateway.yaml`, that contains the following information:
++
+.Example `Gateway` CR
+[source,yaml]
+----
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: example-gateway
+  namespace: openshift-ingress <1>
+spec:
+  gatewayClassName: openshift-default <2>
+  listeners:
+  - name: https <3>
+    hostname: "*.gwapi.${DOMAIN}" <4>
+    port: 443
+    protocol: HTTPS
+    tls:
+      mode: Terminate
+      certificateRefs:
+      - name: gwapi-wildcard <5>
+    allowedRoutes:
+      namespaces:
+        from: All
+----
+<1> The `Gateway` object must be created in the `openshift-ingress` namespace.
+<2> The `Gateway` object must reference the name of the previously created `GatewayClass` object.
+<3> The HTTPS listener listens for HTTPS requests that match a subdomain of the cluster domain. You use this listener to configure ingress to your applications by using Gateway API `HTTPRoute` resources.
+<4> The hostname must be a subdomain of the Ingress Operator domain. If you use a domain, the listener tries to serve all traffic in that domain.
+<5> The name of the previously created secret using the default certificate.
+
+.. Apply the resource by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f example-gateway.yaml
+----
+
+.. Optional: When you create a `Gateway` object, {SMProductName} automatically provisions a deployment and service with the same name. Verify this by running the following commands:
++
+[source,terminal]
+----
+$ oc get deployment -n openshift-ingress example-gateway
+----
++
+[source,terminal]
+----
+$ oc get service -n openshift-ingress example-gateway
+----
++
+.Example output
+[source,terminal]
+----
+NAME              READY   UP-TO-DATE   AVAILABLE   AGE
+example-gateway   1/1     1            1           25s
+----
++
+.Example output
+[source,terminal]
+----
+NAME              TYPE           CLUSTER-IP   EXTERNAL-IP         PORT(S)      AGE
+example-gateway   LoadBalancer   10.1.2.3     <external_ipname>   <port_info>  47s
+----
+
+.. Optional: The Ingress Operator automatically creates a `DNSRecord` CR using the hostname from the listeners, and adds the label `istio.io/gateway-name=example-gateway`. Verify the status of the DNS record by running the following command:
++
+[source,terminal]
+----
+$ oc get dnsrecord -n openshift-ingress -l istio.io/gateway-name=example-gateway -o yaml
```

Comment:
https://github.com/openshift/openshift-docs/pull/94480/files#r2145462579 should be here.